protocol/src/protocol_generated.rs

```diff
@@ -101,14 +102,14 @@ const ENUM_NAMES_SYNC_PAYLOAD:[&'static str; 9] = [
 ];
 
 pub fn enum_name_sync_payload(e: SyncPayload) -> &'static str {
-  let index: usize = e as usize;
-  ENUM_NAMES_SYNC_PAYLOAD[index]
+  let index = e as u8;
+  ENUM_NAMES_SYNC_PAYLOAD[index as usize]
 }
 
 pub struct SyncPayloadUnionTableOffset {}
 #[allow(non_camel_case_types)]
 #[repr(u8)]
-#[derive(Clone, Copy, PartialEq, Debug)]
+#[derive(Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash, Debug)]
```

Comment:
Do you use a new version flatbuffer generator?

Reply:
```
$ flatc --version
flatc version 1.11.0
```
It should have been upgraded some time ago, although I don’t remember when it was